docs/source/guides/contributor/chapters/plugins.rst

```diff
@@ -183,6 +183,35 @@ has to be executed early in the code so try to keep the required deps
 minimal (for example the `avocado.core.settings.settings` is not yet
 available).
 
+Plugins execution order
+=======================
+Avocado execute plugins serially one after each other. Because of that, 
+there are cases where the order of executions is important. For example, 
+package plugin installs necessary dependencies and sysinfo plugin collects 
+system information to check if the dependencies were installed. To ensure that 
+package plugin will be executed before sysinfo plugin, avocado is introducing 
+plugin priority.
+
+The plugins with higher priority will be executed earlier than the plugins with 
+lower priority. The priorities are on the scale between 1-100 and as default 
```

Comment:
But above, you used priority = 0 for TAP plugin, right? Is 0 supported or not?

Reply:
Actually, the TAP resolver is a very specific resolver which never should be executed by dispatcher. Because we can't decide automatically if something is TAP test without running the test and see the results. It is just for usage by Job API.  So that is why I decided to set priority = 0 which is out of the scope from documentation. With this, the TAP resolver will be always at the end of the list. And that is desired behavior.

Reply:
Maybe then introduce a `PRIORITY_LAST_RESORT` kind of symbol with value `0` for consistency?

Reply:
Yes, but regardless of the specificity of a plugin.... I was talking about the inconsistency between the sentence "The priorities are on the scale between 1-100..." and the actual code. This is still confusing. Maybe we could use "The priorities are on the scale between 0-100"